src/Umbraco.Cms.Targets/buildTransitive/Umbraco.Cms.Targets.targets

```diff
@@ -1,20 +1,46 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Target Name="CopyAppsettingsSchema" BeforeTargets="Build">
+  <!-- Automatically nest files (this needs to be in this file, because the props file is imported too early) -->
+  <ItemGroup>
+    <Content Update="appsettings.*.json">
+      <DependentUpon>appsettings.json</DependentUpon>
+    </Content>
+    <Content Update="appsettings-schema.*.json">
+      <DependentUpon>appsettings-schema.json</DependentUpon>
+    </Content>
+  </ItemGroup>
+  
+  <!-- Copy JSON schema files into the project directory -->
+  <Target Name="CopyUmbracoJsonSchemaFiles" BeforeTargets="Build">
+    <Message Text="Copying JSON schema files into project directory: @(UmbracoJsonSchemaFiles->'%(Filename)%(Extension)')" Importance="high" />
+    <Copy SourceFiles="@(UmbracoJsonSchemaFiles)" DestinationFolder="$(MSBuildProjectDirectory)" SkipUnchangedFiles="true" />
+  </Target>
+
+  <!-- Remove copied JSON schema files from the project directory -->
+  <Target Name="RemoveUmbracoJsonSchemaFiles" BeforeTargets="Clean">
+    <Message Text="Removing JSON schema files from project directory: @(UmbracoJsonSchemaFiles->'%(Filename)%(Extension)')" Importance="high" />
+    <Delete Files="@(UmbracoJsonSchemaFiles->'$(MSBuildProjectDirectory)\%(Filename)%(Extension)')" />
+  </Target>
+
```

Comment:
I'm not 100% sure if we should delete the JSON schema files that are copied into the project directory on clean, as they will be overwritten (if changed) on build anyway... This might make more sense if the reference is also removed from the `appsettings-schema.json` file, but that would cause a lot of changes (and disk I/O) when doing rebuilds.

Reply:
While testing, I noticed errors while trying to copy these files into the output directory when doing a rebuild. Although that can probably be fixed by ensuring this target runs before `CopyUmbracoJsonSchemaFiles' or also removing them from the `_SourceItemsToCopyToOutputDirectory` internal MSBuild items, removing this target seems the safest and least hacky...